### PR TITLE
[BST-12096] Create new secrets rule database

### PR DESCRIPTION
**Pull request by nlipsyc**
Create rule for each original rule ID.

Script to generate this is in the ticket.

### CODE REVIEW

**Comment by JonathanSerafini** [comment]
not sure what's up the the pr ... there's a bunch of unrelated commits, don't generate a diff though

File: rules-realm/boostsecurityio/stored-secrets/rules.yaml [hunk 1]
```diff
@@ -1,11 +1,3472 @@
 rules:
-  CWE-200:
-    name: CWE-200
-    pretty_name: CWE-200 - Exposure of Sensitive Information to an Unauthorized Actor
+  CWE-200-0:
```
[the remark this answers is not in the document]
Comment (nlipsyc) [reply rules-realm/boostsecurityio/stored-secrets/rules.yaml:2]: Ahh, is that what was meant by "secret type" here? Got it.

> it categorizes better by using the secret type as rule id.

**Comment by nlipsyc** [comment]
> not sure what's up the the pr ... there's a bunch of unrelated commits, don't generate a diff though

That's from merging in main. The PR  was accidentally pointed at the original repo, not the fork, so the commit hashes match prod instead of dev. I can rebase if it's an issue but I figured since we're squashing it doesn't matter.

File: rules-realm/boostsecurityio/stored-secrets/rules.yaml [hunk 2]
```diff
@@ -1,11 +1,3492 @@
...
   CWE-200:
     name: CWE-200
-    pretty_name: CWE-200 - Exposure of Sensitive Information to an Unauthorized Actor
+    pretty_name: Exposed Secret - CWE-200
     group: stored-secrets
     ref: https://cwe.mitre.org/data/definitions/200.html
-    description: The repository exposes sensitive information to an actor that is
-      not explicitly authorized to have access to that information.
+    description: The repository exposes sensitive information (Cwe 200)
+      to an actor that is not explicitly authorized to have access to that information.
```
Comment (ledo01) [inline rules-realm/boostsecurityio/stored-secrets/rules.yaml:8]: Not sure if we should change the pretty name & description of this rule, it's currently in use by gitleaks.

**Review by ledo01 — APPROVED** [review]
Looks mostly right to me, I think it would be best to avoid changing the existing CWE-200 rule, but maybe that was planned, and I am not aware of it.

File: rules-realm/boostsecurityio/stored-secrets/rules.yaml (hunk 2 above)
Comment (nlipsyc) [reply rules-realm/boostsecurityio/stored-secrets/rules.yaml:8]: I wasn't sure about that. There was a req to not impact existing rules, but I wasn't clear on the mechanics of that (if we just needed to keep the ID the same). Should I switch it back?

**Review by ledo01 — APPROVED** [review]

**Pull request merged**